wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder+Proteus.swift

```diff
@@ -32,7 +32,7 @@ extension EventDecoder {
         in context: NSManagedObjectContext,
         using decryptFunction: ProteusDecryptionFunction
     ) async -> ZMUpdateEvent? {
-        let eventAttributes = [LogAttributesKey.eventId.rawValue: event.uuid?.safeForLoggingDescription ?? "<not filled>"]
+        let eventAttributes: LogAttributes = [LogAttributesKey.eventId.rawValue: event.uuid?.safeForLoggingDescription ?? "<not filled>"].merging(LogAttributes.safePublic, uniquingKeysWith: { _, new in new })
         WireLogger.updateEvent.info("decrypting proteus event...", attributes: eventAttributes)
```

Comment:
What do you think to offer a variadic parameter with zero or more parameters?
The call would look like:
```swift
        WireLogger.updateEvent.info(
            "decrypting proteus event...",
            attributes: [
                LogAttributesKey.eventId.rawValue: event.uuid?.safeForLoggingDescription ?? "<not filled>"
            ],
            .safePublic
        )
```

This can be archived by adding/or replacing the current func and offer it in the protocol:
```swift
    // protocol
    public func info(_ message: any LogConvertible, attributes: LogAttributes...)

    // implementation
    private func log(
        level: LogLevel,
        message: LogConvertible,
        attributes: [LogAttributes]
    ) {
        var mergedAttributes: LogAttributes = [:]
        attributes.forEach {
            mergedAttributes.merge($0) { _, new in new }
        }
        
        // implementation
    }
```

Reply:
I'm wondering now also if the `.rawValue` can be replaced with the key itself:
`LogAttributes = [LogAttributesKey: Encodable]`

Reply:
Let's do that on develop on dedicated PR